Makefile

```
@@ -0,0 +1,20 @@
+NAMESPACE=test-app
+
+define wait_command
+	@echo Waiting for $(2) for $(3)...
+	@time timeout --foreground $(3) bash -c "until $(1); do echo $(2) not ready yet, trying again in $(4)s...; sleep $(4); done"
+	@echo $(2) ready!
+endef
+
+.PHONY: test
+test:
+	@echo "SUCCESS"
+
+.PHONY: deploy
+deploy:
+	@oc new-project $(NAMESPACE)
```

```suggestion
	-@oc new-project $(NAMESPACE)
```

This supresses an error output if the namespace already exists